### DISCUSSION TITLE
Peculiarity with row level security commands in documentation

### DISCUSSION THREAD

**Discussion by cloudorbush** [supabase/supabase#3616] [Questions] [answered]
Is this command:

```pgsql
CREATE POLICY "Enable update for users based on id" ON users FOR
UPDATE
    WITH CHECK (auth.uid() = id);
```

Equivalent to this command?

```pgsql
CREATE POLICY "Enable update for users based on id" ON users FOR
UPDATE
    USING (auth.uid() = id) WITH CHECK (auth.uid() = id);
```
Just wondering as I encountered both of these in the supabase docs.

I would like to alleviate my anxiety on which one to use. 😊

**Comment by VictorPeralta** [comment, marked as answer]
From the PostgreSQL docs:

The USING expression determines which records the UPDATE command will see to operate against, while the WITH CHECK expression defines which modified rows are allowed to be stored back into the relation

**Reply by cloudorbush** [reply]
thank you!
